spec/models/concerns/outdateable_spec.rb

```diff
@@ -48,20 +48,26 @@
   end
 
   describe '#outdate!' do
+    let(:new_base_path) { '/new/base/path' }
```

Comment:
Please, read my comment above. I'd rather not mixing `base_path` in the `outdateable` concern.